src/AddTodoForm.js

```diff
@@ -16,13 +17,14 @@ function AddTodoForm({ onAddTodo }) {
   }
   return (
     <form onSubmit={handleAddTodo}>
-      <label htmlFor="todoTitle">Title</label>
-      <input
+      <InputWithLabel
         name="title"
         id="todoTitle"
```

Comment:
To answer your question in the assignment submission:
Since `InputWithLabel` is a reusable component, these properties should be dynamically assigned to the component by destructuring them in the component's arguments (props) and applying them to the necessary JSX elements:

src/InputWithLabel.js
```diff
- const InputWithLabel = ({ todoValue, onInputChange, children }) => {
+ const InputWithLabel = ({ children, id, name, onInputChange, todoValue }) => {
  const inputRef = useRef();
  useEffect(() => {
    inputRef.current.focus();
  }, [todoValue]);

  return (
    <>
-     <label htmlFor="todoTitle">{children}</label>
+     <label htmlFor={id}>{children}</label>
      <input
-       id="todoTitle"
+       id={title}
-       name="title"
+       name={name}
        onChange={onInputChange}
        ref={inputRef}
        value={todoValue}
      />
    </>
  );
};

export default InputWithLabel
```